modules/k8s-nmstate-deploying-nmstate-CLI.adoc

```diff
@@ -72,7 +72,22 @@ spec:
 EOF
 ----
 
-. Create instance of the `nmstate` operator:
+. Confirm the `ClusterServiceVersion` (CSV)status for the `nmstate` Operator deployment `Succeeded`:
++
+[source,terminal,subs="attributes+"]
+----
+oc get clusterserviceversion -n openshift-nmstate \
+ -o custom-columns=Name:.metadata.name,Phase:.status.phase
+----
++
+.Example output
```

Comment:
Tedious formatting will be needed to bring the "Phase" heading over the "Succeeded" state.

![Screenshot from 2024-03-08 12-53-36](https://github.com/openshift/openshift-docs/assets/57954076/1cc62d0e-a2f2-44df-b48d-251bfbfff708)